ddl/ddl_api.go

```diff
@@ -2983,6 +2996,11 @@ func checkModifyTypes(origin *types.FieldType, to *types.FieldType, needRewriteC
 	return errors.Trace(err)
 }
 
+func checkDefaultValueIsExpr(option *ast.ColumnOption) (bool, error) {
+	_, isSeqExpr, err := tryToGetSequenceDefaultValue(option)
+	return isSeqExpr, errors.Trace(err)
+}
+
```

Comment:
Why not remove this function, and check it directly in `checkAndCreateNewColumn`?

Reply:
maybe the function name is not suitable in here